lib/models/CartoonCat.js

```diff
@@ -0,0 +1,28 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  lives;
+
```

Comment:
make sure to include all the columns in this space. to ensure they make it into your table